tests/testthat/test-regressionlinear.R

```diff
@@ -360,6 +360,32 @@ test_that("Analysis handles categorical predictors in model summary table", {
                       )
 })
 
+test_that("Part And Partial Correlations table results match", {
```

Comment:
shouldn't this test be in this file https://github.com/jasp-stats/jaspRegression/blob/master/tests/testthat/test-regressionlinearVerification.R?

Reply:
Good question; my impression was that we should not put anything inside of test-verified files unless the test is also inside of jasp-verification-project? So my idea was to put it inside of the other test file once I do https://github.com/jasp-stats/jasp-verification-project/issues/7

I agree though that this somewhat duplicates the work, as the plan is to move the code there eventually anyways. Perhaps @AlexanderLyNL has an opinion about it?

Reply:
In general perhaps there is some opportunity to make adding the verified tests into the verification book more automated

Reply:
> adding the verified tests into the verification book more automated

That would be great. There is a PR to at least update the webpage automatically after each PR though. But right now the verified tests are shown via markdown files (e.g., see https://github.com/jasp-stats/jasp-verification-project/pull/6/files#diff-7c28bc84a09b4ea59bdd8191146496f967d815b2dffd972b123cc30808929a8b). It would be great if we could automate this. We could do something like

1. Move all the markdown files inside https://github.com/jasp-stats/jasp-verification-project/pull/6 to their respective jasp-*** repositories, into some specific subfolder.
2. Add an action that scrapes the jasp-desktop repositories once a week for changes to the files, copy-pastes any changes to jasp-stats/jasp-verification-project, and then rebuild the site.

I'm not sure if/ how those markdown files could simultaneously be used as unit tests though.

Reply:
I suspect that there's a bit of doubling of work currently. The Bookdown and Github Action PR for the jasp-verification-project is ongoing. Currently, Jonas is working on getting the rmd files automatically deploy to the verification website. 

The dream is indeed to have the (1) rmds automatically make unit test, or (2) the unit tests automatically add an entry to the rmd files. But that will be a bit later. So sorry for the current duplication work.

Reply:
> I'm not sure if/ how those markdown files could simultaneously be used as unit tests though.

> The dream is indeed to have the (1) rmds automatically make unit test, or (2) the unit tests automatically add an entry to the rmd files. But that will be a bit later. So sorry for the current duplication work.

Yeah that's the challenge, however going from tests to rmarkdown should be easier no?

So what could happen is:
1. All verified tests will be written in the `test-[analysis]-verified.R` files with some markdown-style comments detailing what should be the results etc.
2. Whenever there is an update to a jaspModule, jasp-verified-project checks out the verified tests R scripts, calls `knitr::spin()` to turn them into .Rmd files, and rebuilds the website.

Of course that would duplicate the work on the other side (turning the original Rmd files into plain .R files), but that also does not need to be done right now (for some time, we could have some chapters in the book in the originally hand written .Rmd files and at the same time generate automatically new chapters from verified tests inside of the respective jaspModules). Another question is whether writing the tests such that they can be turned into nicely formatted .Rmd files is easy.